config/crd/kustomization.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.1
+- github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.5.0-dev
```

Comment:
@clock21am This `v0.5.0-dev` ref doesn't seem to exist. Is this something you expect to exist soon?